controllers/favorites.js

```diff
@@ -0,0 +1,37 @@
+const Favorite = require('../models/Favorite');
+
+module.exports = {
+
+  getFavorites: async (req, res) => {
+    const currentPage = "favorites";
+    const skip =
+      parseInt(req.query.skip || "0", 10) <= 0
+        ? 0
+        : parseInt(req.query.skip, 10);
+    const limit = req.query.limit || 4;
+    console.log(currentPage);
+    try {
+      const totalFavorites = await Favorite.find({
```

Comment:
Very minor formatting suggestion -- these lines could be made to be slightly more symmetrical like this:
```
const totalFavorites = await Favorite.find({ user: req.user.id })
  .countDocuments();
const recipes = await Favorite.find({ user: req.user.id })
  .populate("recipe")
  .skip(skip)
  .limit(limit);```

Reply:
Changed! Making the commits now!